### PR TITLE
HIFI Workshops Page

### PR DESCRIPTION
**Pull request by amlimjoc**
#45

### CODE REVIEW

File: src/images/Solidworks.png
Comment (benson-fm) [inline src/images/Solidworks.png:1]: On my ned it seems the images are lower of quality

File: src/images/WorkshopsBackground.png
Comment (benson-fm) [inline src/images/WorkshopsBackground.png:1]: same thing with lower quality on my end, you might want to download a bigger picture in dimensions, i think you could also do an increase resolution on this image online converter

File: src/pages/ProgramsPage/WorkshopsPage/CheckOut.jsx [hunk 1]
```diff
@@ -0,0 +1,133 @@
...
+    <img className="w-[16.13rem] h-[10rem] object-cover" src={image} alt={title} />
+    <div className="flex flex-col justify-center w-[19.25rem] h-[8.5rem] text-left ml-4">
+      <h3 className="font-[600] text-2xl leading-[1.75rem] tracking-[0.12em] font-karla text-white">{title}</h3>
+      <p className="text-[1rem] leading-[1.5rem] font-inter font-[300] text-white">{description}</p>
+    </div>
+  </div>
+);
+
+// Right side slideshow component
+const Slideshow = ({ slides }) => {
```
Comment (benson-fm) [inline src/pages/ProgramsPage/WorkshopsPage/CheckOut.jsx:20]: great job with this interval mechanic

File: src/pages/ProgramsPage/WorkshopsPage/JoinWorkshops.jsx [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    const ref = useRef();
+    const isVisible = useIsVisible(ref);
+
+    return (
+        <div className="bg-white w-full h-[487.5px] flex items-center justify-center py-72">
+            <div ref={ref} className={`flex flex-col items-center justify-center w-[60rem] h-[30rem] transition-opacity ease-in duration-700 ${isVisible ? "opacity-100" : "opacity-0"}`}>
+                <h1 className="font-karla font-[700] text-5xl text-black pb-3">Boost Your Skills: <span className="text-[#FFA523]">Join Our Workshops</span></h1>
+                <div className="mt-[30px] border-t-4 border-[#FAC80A] w-[10rem]"></div> 
+                <div className="mt-[40px]">
+                    <p className="font-inter text-[16px] leading-8 text-[#3C3C3C    ] text-center">
```
Comment (benson-fm) [inline src/pages/ProgramsPage/WorkshopsPage/JoinWorkshops.jsx]: I don't think the color is registering on my end, its gray on my side

File: src/pages/ProgramsPage/WorkshopsPage/CheckOut.jsx [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+    },
+    {
+      image: Solidworks,
+      title: "Solidworks",
+      description: "BUILD • PROGRAM • TEST",
+    },
+  ];
+
+  return (
+    <div
```
Comment (benson-fm) [inline src/pages/ProgramsPage/WorkshopsPage/CheckOut.jsx:96]: I believe the code s here but lets add some padding to the carousel component just so its not too close to the bottom

**Review by benson-fm — CHANGES_REQUESTED** [review]
Great job Arvin! Literally THE GOAT

File: src/pages/ProgramsPage/WorkshopsPage/JoinWorkshops.jsx (hunk 2 above)
Comment (amlimjoc) [reply src/pages/ProgramsPage/WorkshopsPage/JoinWorkshops.jsx]: This one I'm not sure what you're seeing, it shows on my end
image

**Pull request merged**